### PR TITLE
doc typos

### PR DESCRIPTION
**Pull request by jwkaltz**

### CODE REVIEW

File: doc/integrator/create_application.rst.mako [hunk 1]
```diff
@@ -30,7 +30,7 @@ created by the c2cgeoportal scaffolds (the ``c2cgeoportal_create`` and
 Install c2cgeoportal
 --------------------
 
-This step is required if you cannot, or do not want, to create the c2cgeoportal
+This step is required if you cannot, or do not want to, create the c2cgeoportal
```
Comment (sbrunner) [inline doc/integrator/create_application.rst.mako]: For me line 33 to 42 is obsolete...

File: doc/integrator/install_application.rst [hunk 2]
```diff
@@ -3,21 +3,17 @@
...
 
-For example If you want to use an existing database you should ignore
+If you want to use an existing database, you should ignore
 all the commands concerning the database.
 
-This guide considers that:
- - We use a server manages by Camptocamp, meaning:
-    - all dependencies described in the
-      :ref:`requirements <integrator_requirements>` are installed,
-    - Postgres has a GIS template ``template_postgis`` and a user ``www-data``,
```
Comment (sbrunner) [inline doc/integrator/install_application.rst]: Why removing this line?

Comment (jwkaltz) [reply doc/integrator/install_application.rst]: Because later in the instructions, it says what to do if you don't have template_postgis

Comment (jwkaltz) [reply doc/integrator/install_application.rst]: And it also says what to do if do not want to use "www-data" as user

File: doc/integrator/create_application.rst.mako (hunk 1 above)
Comment (jwkaltz) [reply doc/integrator/create_application.rst.mako]: Indeed - removed, now.

**Pull request merged**